client/assets/stylesheets/layout/_reset.scss

```diff
@@ -0,0 +1,150 @@
+//@mixin calypso-reset {
```

Comment:
We don't _need_ these to be in mixin, but it would make things easier for https://github.com/Automattic/wp-calypso/pull/36170 and consumers outside Calypso.